src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<p>lowercase 't' in 'There'.</p><p> </p><p>lowercase 's' in 'Some'.</p><p> </p><p>exhibit -&gt; have</p><p> </p><p>"distort analysis, prompting their removal." -&gt; "distort analysis. Filter these values out of the data."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>